docs/docs/features/low-power-states.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Low Power States
+sidebar_label: Low Power States
+---
+
+## Idle
+
+In the idle state, peripherals such as displays and lighting are disabled, but the keyboard remains connected to Bluetooth so it can immediately respond when you press a key. Idle state is entered automatically after a timeout period that is [30 seconds by default](../config/power.md#low-power-states).
+
+## Deep Sleep
+
+In the deep sleep state, the keyboard enters a software power-off state. Among others, this:
+
+- Disconnects the keyboard from all Bluetooth connections
+- Disables any peripherals such as displays and lighting
+- If possible, external power output is disabled
+- Clears the contents of RAM, including any unsaved [Studio](studio.md) changes
+
+This state uses very little power, but it may take a few seconds to reconnect after waking. A [wakeup source](#wakeup-sources) is required to wake from deep sleep.
+
+### Config
+
+Deep sleep must be enabled via its corresponding [config](../config/power.md#low-power-states).
+
+### Wakeup Sources
+
+Using deep sleep requires `kscan` nodes to have the `wakeup-source` property to enable them to wake the keyboard, e.g.:
+
+```dts
+/ {
+    kscan: kscan {
+        compatible = "zmk,kscan-gpio-matrix";
+        diode-direction = "col2row";
+        wakeup-source;
+
+        ...
+    };
+};
+```
+
+It is recommended to add the `wakeup-source` property to `kscan` devices even if the deep sleep feature is not used -- there is no downside to it.
```

Comment:
This ideally wouldn't be here, since it is more hardware integration than user-facing. Maybe we can link to the new-shield kscan section, and say "your keyboard should be configured with this by default"?

Reply:
I see your point, but I'd like to leave it here for now. It's aimed towards those whose keyboards are custom and haven't been updated with the property yet. I'm happy to remove it and rephrase once we're reasonably confident that almost everyone affected has modified their keyboard. My estimate would be a few months once we have versioning, Zephyr 3.7, and a few other things.

Reply:
It's been 6+ months and I haven't been seeing a lot of issues related to the missing property lately, but we can leave it if you want to. I hope it doesn't cause more confusion than it solves.